test/integration/integration_admin_test.cc

```diff
@@ -342,17 +350,27 @@ TEST_P(IntegrationAdminTest, Admin) {
                                                 downstreamProtocol(), version_);
   EXPECT_TRUE(response->complete());
   EXPECT_EQ("200", response->headers().Status()->value().getStringView());
+  EXPECT_EQ("text/plain; charset=UTF-8", ContentType(response));
+
+  response = IntegrationUtil::makeSingleRequest(
+      lookupPort("admin"), "GET", "/listeners?format=json", "", downstreamProtocol(), version_);
```

Comment:
can we add a test that validates the text format?

Reply:
done